carveracontroller/main.py

```diff
@@ -2605,13 +2609,20 @@ def setUIForModel(self, model, *args):
         app = App.get_running_app()
         if model != app.model:
             app.model = model.strip()
-            if app.model == 'CA1':
-                self.tool_drop_down.set_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',  
-                                                           'Tool: 6', 'Laser', 'Custom']
-                self.tool_drop_down.change_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
-                                                              'Tool: 5', 'Tool: 6', 'Laser', 'Custom']
-                CNC.vars['rotation_base_width'] = 300
-                CNC.vars['rotation_head_width'] = 38
+        if app.model == 'CA1':
+            self.tool_drop_down.set_dropdown.values = ['Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',
+                                                        'Tool: 6', 'Laser', 'Custom']
+            self.tool_drop_down.change_dropdown.values = ['Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
```

Comment:
same as https://github.com/Carvera-Community/Carvera_Controller/pull/263/files#r2171057440

```suggestion
            self.tool_drop_down.change_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
```